nautobot/docs/configuration/required-settings.md

```diff
@@ -357,6 +357,13 @@ $ nautobot-server generate_secret_key
 +$_kw69oq&fbkfk6&q-+ksbgzw1&061ghw%420u3(wen54w(m
 ```
 
+Alternatively use the following command to generate a secret even before `nautobot-server` is runable:
+
+```no-highlight
+$ tr -cd '[=CHAR=abcdefghijklmnopqrstuvwxyzABCDEFGHIJKLMNOPQRSTUVWXYZ0123456789!@#$%^&*(-_=+)]' < /dev/urandom | fold -w50 | head -n1
```

Comment:
On my Mac this fails with `tr: misplaced sequence asterisk`, probably because the Mac uses the BSD `tr` variant. I know we don't formally support deployment on Mac but broader compatibility is still desirable I think. After some experimentation, the below alternative appears to work both on my Mac and in Debian under Docker:

```suggestion
$ LC_ALL=C tr -cd '[:lower:][:digit:]!@#$%^&*(\-_=+)' < /dev/urandom | fold -w50 | head -n1
```